### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by AlexeyMaslyvskyi**
done

### CODE REVIEW

File: src/partials/footer.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<footer class="footer">
+  <div class="conteiner">
+    <div class="footer-box">
+      <div class="footer-box__name">
+        <h2 class="footer-box__head">homemade ice cream made</h2>
+        <p class="footer-box__head--description">with passion</p>
+      </div>
+      <div class="footer-box__contacts">
+        <ul class="footer-box__contacts-list list">
```
Comment (ruudyaa) [inline src/partials/footer.html]: the class "list" is unnecessary, styles for <ul> were reset in reset.scss

File: src/partials/footer.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<footer class="footer">
+  <div class="conteiner">
+    <div class="footer-box">
+      <div class="footer-box__name">
+        <h2 class="footer-box__head">homemade ice cream made</h2>
+        <p class="footer-box__head--description">with passion</p>
```
Comment (ruudyaa) [inline src/partials/footer.html]: should be "footer-box__description"
--class - modificator, used for styles

File: src/partials/footer.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      </div>
+      <div class="footer-box__contacts">
```
Comment (ruudyaa) [inline src/partials/footer.html]: better to use a separate class "footer-contacts"

File: src/partials/footer.html (hunk 1 above)
Comment (ruudyaa) [reply src/partials/footer.html]: rename "footer-contacts__list"

File: src/partials/footer.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        <ul class="footer-box__contacts-list list">
+          <li class="footer-box__contacts--item">
```
Comment (ruudyaa) [inline src/partials/footer.html]: rename "footer-contacts__item"

File: src/partials/footer.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            <a
+              class="footer-box__contacts--item-link link"
```
Comment (ruudyaa) [inline src/partials/footer.html]: rename "footer-contacts__link"
delete class "link"

File: src/partials/footer.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+              href="https://github.com/"
+              ><svg
+                class="footer-box__contacts--item-icon"
```
Comment (ruudyaa) [inline src/partials/footer.html]: rename "footer-contacts__icon"

File: src/partials/footer.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+                width="20px"
+                height="20px"
+              >
+                <use href="./images/icons.svg#icon-github"></use></svg
+            ></a>
+          </li>
+          <li class="footer-box__contacts--item">
```
Comment (ruudyaa) [inline src/partials/footer.html]: "footer-contacts__item"

File: src/partials/footer.html [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+            <a
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer-contacts__link

File: src/partials/footer.html [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+              class="footer-box__contacts--item-link link"
```
Comment (ruudyaa) [inline src/partials/footer.html]: delete link

File: src/partials/footer.html [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+              href="https://web.telegram.org/k/"
+              ><svg
+                class="footer-box__contacts--item-icon"
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer-contacts__icon

File: src/partials/footer.html [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+                width="20px"
+                height="20px"
+              >
+                <use href="./images/icons.svg#icon-telegram"></use></svg
+            ></a>
+          </li>
+          <li class="footer-box__contacts--item">
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer-contacts__item

File: src/partials/footer.html [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+            <a
+              class="footer-box__contacts--item-link link"
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer-contacts__link

File: src/partials/footer.html [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+              href="https://uk-ua.facebook.com/"
+              ><svg
+                class="footer-box__contacts--item-icon"
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer-contacts__icon

File: src/partials/footer.html [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+                width="20px"
+                height="20px"
+              >
+                <use href="./images/icons.svg#icon-facebook"></use></svg
+            ></a>
+          </li>
+        </ul>
+        <p class="footer-box__call">
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer__call

Comment (ruudyaa) [reply src/partials/footer.html]: footer-call

Comment (ruudyaa) [reply src/partials/footer.html]: use footer-call

File: src/partials/footer.html [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+          Feel free to contact
+          <a class="footer-box__call--link link" href="tel: +610383766284"
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer-call__link 
class "link" delete

File: src/partials/footer.html [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+            >+61 (0) 383 766 284</a
+          >
+        </p>
+      </div>
+    </div>
+
+    <div class="footer__copyrights">
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer-copyrights

File: src/partials/footer.html [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+      <p class="footer__copyrights--item">&copy; 2022 - IceCream.</p>
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer-copyrights__item

File: src/partials/footer.html [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+      <p class="footer__copyrights--item">All Rights Reserved.</p>
```
Comment (ruudyaa) [inline src/partials/footer.html]: footer-copyrights__item

File: src/sass/layout/_footer.scss [hunk 19]
```diff
@@ -0,0 +1,208 @@
+.footer {
+  background-color: getColor('green-main');
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary. it's written for <body>.

Comment (AlexeyMaslyvskyi) [reply src/sass/layout/_footer.scss:2]: what about paddings?
also delete?

Comment (AlexeyMaslyvskyi) [reply src/sass/layout/_footer.scss:2]: or just bgc?

File: src/sass/layout/_footer.scss [hunk 20]
```diff
@@ -0,0 +1,208 @@
...
+  padding: 0 20px 24px 20px;
+  @media (min-width: 768px) {
+    padding: 0;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: padding: 0 0 12px 0;

File: src/sass/layout/_footer.scss [hunk 21]
```diff
@@ -0,0 +1,208 @@
+.footer {
+  background-color: getColor('green-main');
+  padding: 0 20px 24px 20px;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss:2]: @media (min-width: 480px) {
    padding: 0 0 24 0;

File: src/sass/layout/_footer.scss [hunk 22]
```diff
@@ -0,0 +1,208 @@
...
+  @media (min-width: 768px) {
+    padding: 0;
+  }
+  @media (min-width: 1200px) {
+    padding: 0 16px 0 16px;
+  }
+}
+
+.footer-box {
+  @media (min-width: 768px) {
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss:12]: should be there padding-bottom as for the desktop version?
for tablet - 32px;
for 480px and mobile - 40px

File: src/sass/layout/_footer.scss [hunk 23]
```diff
@@ -0,0 +1,208 @@
...
+    width: 352px;
+  }
+  @media (min-width: 1200px) {
+    width: 504px;
+  }
+}
+.footer-box__head {
+  display: inline-block;
+  margin: 0;
+  font-family: 'Montserrat';
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary.

File: src/sass/layout/_footer.scss [hunk 24]
```diff
@@ -0,0 +1,208 @@
...
+  font-style: normal;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary.

File: src/sass/layout/_footer.scss [hunk 25]
```diff
@@ -0,0 +1,208 @@
...
+    font-size: 30px;
+    line-height: 37px;
+    margin-bottom: 14px;
+  }
+}
+.footer-box__head--description {
+  margin: 0;
+  margin-bottom: 60px;
+  display: inline-block;
+  font-family: 'Montserrat';
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary

File: src/sass/layout/_footer.scss [hunk 26]
```diff
@@ -0,0 +1,208 @@
...
+  font-style: normal;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary

File: src/sass/layout/_footer.scss (hunk 22 above)
Comment (AlexeyMaslyvskyi) [reply src/sass/layout/_footer.scss:12]: there are some margins

File: src/sass/layout/_footer.scss [hunk 27]
```diff
@@ -0,0 +1,208 @@
...
+  justify-content: end;
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  margin-bottom: 20px;
+  @media (min-width: 768px) {
+    margin-bottom: 34px;
+    gap: 30px;
+  }
+}
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss:103]: @media (min-width: 1200px) {
    margin-bottom: 40px;
}

File: src/sass/layout/_footer.scss [hunk 28]
```diff
@@ -0,0 +1,208 @@
...
+  width: 34px;
+  height: 34px;
+  fill: getColor('green-main');
+  border-radius: 50%;
+  &:hover {
+    fill: getColor('theme-white');
+  }
+  &:focus-within {
+    fill: getColor('theme-white');
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary shadow

File: src/sass/layout/_footer.scss [hunk 29]
```diff
@@ -0,0 +1,208 @@
...
+  }
+}
+
+.footer-box__call {
+  margin: 0;
+  text-align: right;
+  font-family: 'Montserrat';
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary.

File: src/sass/layout/_footer.scss [hunk 30]
```diff
@@ -0,0 +1,208 @@
...
+  font-style: normal;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary.

File: src/sass/layout/_footer.scss [hunk 31]
```diff
@@ -0,0 +1,208 @@
...
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 186%;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: line-height: calc( 22 / 12) ;

File: src/sass/layout/_footer.scss [hunk 32]
```diff
@@ -0,0 +1,208 @@
...
+
+  color: getColor('theme-black');
+
+  margin-bottom: 40px;
+  @media (min-width: 768px) {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 186%;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: line-height: calc( 26 / 14) ;

File: src/sass/layout/_footer.scss [hunk 33]
```diff
@@ -0,0 +1,208 @@
...
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 186%;
+
+  color: getColor('theme-black');
+
+  margin-bottom: 40px;
+  @media (min-width: 768px) {
+    font-family: 'Montserrat';
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary

File: src/sass/layout/_footer.scss [hunk 34]
```diff
@@ -0,0 +1,208 @@
...
+    font-style: normal;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary

File: src/sass/layout/_footer.scss [hunk 35]
```diff
@@ -0,0 +1,208 @@
...
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 186%;
+  }
+  @media (min-width: 1200px) {
+    margin-bottom: 0;
+
+    font-size: 16px;
+
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss:148]: line-height: calc( 30 / 16);

File: src/sass/layout/_footer.scss [hunk 36]
```diff
@@ -0,0 +1,208 @@
...
+    letter-spacing: 0.02em;
+  }
+}
+.footer-box__call--link {
+  font-family: 'Montserrat';
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary

File: src/sass/layout/_footer.scss [hunk 37]
```diff
@@ -0,0 +1,208 @@
...
+  font-style: normal;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary

File: src/sass/layout/_footer.scss [hunk 38]
```diff
@@ -0,0 +1,208 @@
...
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 186%;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: line-height: calc( 22 / 16);

File: src/sass/layout/_footer.scss [hunk 39]
```diff
@@ -0,0 +1,208 @@
...
+
+  color: getColor('theme-white');
+}
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss:167]: @media (min-width: 768px) {
  font-size: 14px;
  line-height: calc( 26 / 14);
}

  @media (min-width: 1200px) {
  font-size: 16px;
  line-height: calc( 30 / 16);
}

File: src/sass/layout/_footer.scss [hunk 40]
```diff
@@ -0,0 +1,208 @@
...
+.footer__copyrights {
+  margin: 0 auto;
+  text-align: center;
+  padding-top: 20px;
+  border-top: 1px solid #e1e1e1;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: color should be variable

File: src/sass/layout/_footer.scss [hunk 41]
```diff
@@ -0,0 +1,208 @@
...
+    padding: 12px 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+}
+.footer__copyrights--item {
+  margin: 0;
+  font-family: 'Montserrat';
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary

File: src/sass/layout/_footer.scss [hunk 42]
```diff
@@ -0,0 +1,208 @@
...
+  font-style: normal;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: unnecessary

File: src/sass/layout/_footer.scss [hunk 43]
```diff
@@ -0,0 +1,208 @@
...
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 152%;
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: line-height: calc( 21 / 14);

File: src/sass/layout/_footer.scss [hunk 44]
```diff
@@ -0,0 +1,208 @@
...
+
+  color: getColor('footer-grey');
+  @media (min-width: 768px) {
+    margin-left: 5px;
+
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss:189]: line-height: calc(26 / 14);

File: src/sass/layout/_footer.scss [hunk 45]
```diff
@@ -0,0 +1,208 @@
...
+    line-height: 186%;
+  }
+  @media (min-width: 1200px) {
+    font-size: 12px;
+
```
Comment (ruudyaa) [inline src/sass/layout/_footer.scss]: line-height: calc(22 / 12);

**Pull request closed without merging**